rct229/rulesets/ashrae9012019/section5/section5rule28.py

```diff
@@ -101,10 +100,16 @@ def manual_check_required(self, context, calc_vals=None, data=None):
             return all(
```

Comment:
This is one of the convention.
In RDS, the `Rule Assertion - Component` case 1 says For each subsurface, if roof is exterior mixed type and the baseline requirements for residential and non-residential type SHGC for different skylight-roof-ratio are different.
In that case, we perform the `manual_check_required` at each subsurface level. 

So we will need to create a `manual_review_flag` in `create_data` follows this logic at the building level.
Then in the sub-surface level, we take out the `manual_review_flag` and combine with the roof conditioned type for manual check.

One example is [5-20](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/rulesets/ashrae9012019/section5/section5rule20.py). I think in 5-20, it put the manual check logic at AboveGradeWall level. I will move that to subsurface level as well.